web-components/src/components/sliders/ResponsiveSlider.tsx

```diff
@@ -238,8 +246,7 @@ export default function ResponsiveSlider({
         ) : null}
         {items.length > 1 && arrows && desktop && (
           <Grid
-            xs={12}
-            sm={4}
+            xs={renderTitle || title ? 4 : 12}
```

Comment:
I'm confused about why we sent `renderTitle`, a function of type `() => JSX.Element` to this prop that seems to accept `number | bool` ?

Reply:
we set `xs` to 4 or 12 in this case

Reply:
I thought the `||` operator would take priority over the ternary but apparently, that's not the case, all good then.